pkg/controllers/subnetport/subnetport_controller.go

```diff
@@ -296,6 +308,10 @@ func (r *SubnetPortReconciler) SetupFieldIndexers(mgr ctrl.Manager) error {
 	if err := mgr.GetFieldIndexer().IndexField(context.TODO(), &v1alpha1.AddressBinding{}, util.AddressBindingNamespaceVMIndexKey, addressBindingNamespaceVMIndexFunc); err != nil {
 		return err
 	}
+	if err := mgr.GetFieldIndexer().IndexField(context.TODO(), &v1alpha1.SubnetPort{}, "spec.subnet", subnetPortSubnetIndexFunc); err != nil {
```

Comment:
what's different with context.Background() and context.TODO()?

Reply:
context.Background() is usually at the top of program being used in main
context.TODO is a placeholder for when not sure which context to use